###### ROS Bag Recording Parameters ######
# Recorded Topics
recorded_topics: ['UAV1/UAV1_left', 'UAV1/UAV1_right', 'UAV2/UAV2_left', 'UAV2/UAV2_right', 'footIMU/IMU', 'imu_odometry', 'camera_1/odom/sample', 'camera_2/odom/sample', 'command', 'HumanPose', 'UAV1Pose', 'UAV2Pose', 'borealis/command/pose', 'uav1/follow_me_target_odom', 'uav1/gun_target_odom', 'uav1/target_odom', 'uav1/odom', 'uav2/odom']
# Saved Directory
saved_directory: "/home/achala/Bag3"


###### Target Publisher Parameters  ######
# FollowMe / Go Command Topic
command_topic: '/command'
# Commands
follow_me_cmd: 'Follow'
go_cmd: 'Go'

